./install.sh error. - [Makefile:25: build] Error 17  reNgine installation failed!!

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I get this while doing the ./install.sh script:
failed to solve: executor failed running [/bin/sh -c go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest]: exit code: 1
make: *** [Makefile:25: build] Error 17

reNgine installation failed!!

### Expected Behavior

To install correctly. lolol

### Steps To Reproduce

cd rengine
./install.sh

### Environment

```markdown
- reNgine: Latest.
- OS: Kali.
- Python: Latest.
- Docker Engine: Last.
- Docker Compose: Latest.
- Browser: N/A.
```


### Anything else?

_No response_

--- a/web/Dockerfile
+++ b/web/Dockerfile
@@ -48,9 +48,9 @@ RUN apt install -y  --no-install-recommends \
 RUN add-apt-repository ppa:mozillateam/ppa
 
 # Download and install go 1.20
-RUN wget https://golang.org/dl/go1.20.linux-amd64.tar.gz
-RUN tar -xvf go1.20.linux-amd64.tar.gz
-RUN rm go1.20.linux-amd64.tar.gz
+RUN wget https://golang.org/dl/go1.21.4.linux-amd64.tar.gz
+RUN tar -xvf go1.21.4.linux-amd64.tar.gz
+RUN rm go1.21.4.linux-amd64.tar.gz
 RUN mv go /usr/local
 
 # Download geckodriver
@@ -74,7 +74,7 @@ RUN go install -v github.com/tomnomnom/unfurl@latest
 RUN go install -v github.com/tomnomnom/waybackurls@latest
 RUN go install -v github.com/projectdiscovery/httpx/cmd/httpx@latest
 RUN go install -v github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest
-RUN go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest
+RUN go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest
 RUN go install -v github.com/projectdiscovery/naabu/v2/cmd/naabu@latest
 RUN go install -v github.com/hakluke/hakrawler@latest
 RUN go install -v github.com/lc/gau/v2/cmd/gau@latest

--- a/web/dashboard/views.py
+++ b/web/dashboard/views.py
@@ -368,13 +368,13 @@ def onboarding(request):
 
     context['error'] = error
     # check is any projects exists, then redirect to project list else onboarding
-    projects = Project.objects.all()
+    project = Project.objects.first()
 
     context['openai_key'] = OpenAiAPIKey.objects.first()
     context['netlas_key'] = NetlasAPIKey.objects.first()
 
-    if len(projects):
-        slug = projects[0].slug
+    if projects:
+        slug = project.slug
         return HttpResponseRedirect(reverse('dashboardIndex', kwargs={'slug': slug}))
 
     return render(request, 'dashboard/onboarding.html', context)

--- a/web/scanEngine/templates/scanEngine/lookup.html
+++ b/web/scanEngine/templates/scanEngine/lookup.html
@@ -47,7 +47,7 @@ Interesting Lookup
           {{ form.keywords }}
           {# hidden value #}
           {{ form.custom_type }}
-          <span class="text-danger">Press comma , to separate the keywords.</span>
+          <span class="text-muted">Please use comma , to separate the keywords.</span>
           <h4 class=" header-title mt-3">Lookup in</h4>
           <div class="form-check mb-2 form-check-primary">
             {{form.url_lookup}}

--- a/web/scanEngine/templates/scanEngine/lookup.html
+++ b/web/scanEngine/templates/scanEngine/lookup.html
@@ -47,7 +47,7 @@ Interesting Lookup
           {{ form.keywords }}
           {# hidden value #}
           {{ form.custom_type }}
-          <span class="text-muted">Please use comma , to separate the keywords.</span>
+          <span class="text-muted">Please use a comma (,) to separate the keywords.</span>
           <h4 class=" header-title mt-3">Lookup in</h4>
           <div class="form-check mb-2 form-check-primary">
             {{form.url_lookup}}

